lib/tools.js

```diff
@@ -236,6 +237,24 @@ api.clone = function(value) {
 };
 
 /**
+ * Creates a getter for a config property.
```

Comment:
`Assigns a getter to the value of a config key. When the config key is read, the getter will execute and compute the configured value. This is particularly useful for config values that depend on other config values; it removes the need to reset such a value when its dependencies change.`